p11-kit/add-profile.c

```diff
@@ -150,7 +152,7 @@ add_profile (const char *token_str,
 		session = p11_kit_iter_get_session (iter);
 		rv = module->C_CreateObject (session, template, template_len, &object);
 		if (rv != CKR_OK) {
-			p11_message (_("failed to create the profile object: %s"), p11_kit_strerror (rv));
+			p11_message (_("failed to create profile object: %s"), p11_kit_strerror (rv));
```

Comment:
Wouldn't it be more sensible to return an error if there are no matching writable tokens?

Reply:
that reminds me that I should change it to iterate only the tokens and open the session manually